### DISCUSSION TITLE
Filter by assigned to me

### DISCUSSION THREAD

**Discussion by revmischa** [community/community#9667] [Projects and Issues] [answered]
I want a project-wide view where users can look at stuff assigned to just them

**Comment by parisk** [comment]
That's a great idea. It's pretty much the way `@me` works in regular issues[^1]. 

**EDIT**: What's originally mentioned in https://github.com/github/feedback/discussions/5769.

[^1]: https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests

**Comment by azenMatt** [comment, marked as answer]
Just updated #5769 that we shipped this! 

![image](https://user-images.githubusercontent.com/7584089/154979634-365d34e0-eef4-4fca-a49b-fd47b194574d.png)

**Comment by revmischa** [comment]
Sick, thanks!
